config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import "github.com/santhosh-tekuri/jsonschema/v6"
+
+// ValidationOptions A container for validation configuration.
+//
+// Generally fluent With... style functions are used to establish the desired behavior.
+type ValidationOptions struct {
+	RegexEngine jsonschema.RegexpEngine
+}
+
+// Option Enables an 'Options pattern' approach
+type Option func(*ValidationOptions)
+
+// NewOptions creates a new ValidationOptions instance with default values.
+func NewOptions(opts ...Option) *ValidationOptions {
```

Comment:
This will make more sense

```suggestion
func NewValidationOptions(opts ...Option) *ValidationOptions {
```

Reply:
Ack